packages/devtools_app/test/memory_screen_test.dart

```diff
@@ -58,6 +57,7 @@ void main() {
     expect(find.byType(MemoryBody), findsOneWidget);
   }
 
+  // Set a wide enough screen width that we do not run into overflow.
```

Comment:
move unrelated cleanup like this to a separate CL so this main one is easier to review. Code reviews are generally O(n^2) in the number of lines and files to review so if we can break this into a few <2000 line CLs instead of one 10,000 line CL that will help.